XpressAI/xircuits
# 🔋 Enable Toggle Link Animation

# Description
This PR introduces a button to toggle the link animation, which is useful for low powered devices.
It is implemented  as a `powerModeState` by modifying the `Path` property with the global.css. The state is persistent across canvases. 

Additionally the link classes are refactored to better reflect their functions. 

## Pull Request Type


## Type of Change


# Tests

1. Test Toggle Link Animation Feature
    1. Open a workflow and try the toolbar toggle animation button.
    2. Verify that the link animation stops.
    3. Open another workflow. Verify that the animation is also halted on both.
    4. Toggle the animation button again. Verify that it moves again.
2. Test refactors
    1. Simply run and compile a workflow. Verify that it still works.

## Tested on?


# Notes
